roscala/src/main/scala/coop/rchain/rosette/prim/tuple.scala

```diff
@@ -287,6 +287,41 @@ object tuple {
     }
   }
 
+  object tplMemQ extends Prim {
+    override val name: String = "tuple-mem?"
+    override val minArgs: Int = 2
+    override val maxArgs: Int = 2
+
+    @checkArgumentMismatch
+    override def fn(ctxt: Ctxt): Either[PrimError, RblBool] = {
+      val elem  = ctxt.argvec.elem
+      val nargs = ctxt.nargs
+
+      checkTuple(0, elem).map { t =>
+        for (el <- t.elem) {
```

Comment:
Would be nice if you could rewrite this part to not use `return` and a for-loop. Should be doable with `find`.

Reply:
I knew you wouldn't like that construct, but for the life of me, I couldn't get the syntax right last week.  Thanks for the encouragement.  Today it just worked.

Done.